### DISCUSSION TITLE
Welcome to OverPutney Discussions!

### DISCUSSION THREAD

**Discussion by shbisson** [shbisson/OverPutney#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chilledgecko** [comment]
Hi,
I'm an old user of the original Tustin code back in the day.
On returning to it I realised it was no longer actively supported and had been broken by various updates over the years.
I was pleased to find the Putney version which I could adapt to my local requirements.
I'm close to the SEN LCY LHR flightpaths so I've adapted the remarks code to reflect local flight path actions.

ATB

Kevin

**Comment by derektgardner** [comment]
Hello!

I'm a relatively new ADS-B enthusiast. I tried to get the original AboveTustin code working on my machine about a year ago, but my python knowledge was non-existent. That knowledge has only improved slightly, but I'm glad to see that this code is working. I've forked and made a few small changes to get things working on my machine in a way that works for me. 

Thanks for getting this working @shbisson!

-Derek

**Comment by AvioGuy** [comment]
Are there any installation instructions anywhere?
Thanks

**Comment by ggoodloff** [comment]
I cannot get any images to load... Any idea why? 

Also my startup script  states  'source bin/activate'.  <<<---- what is this?
